### DISCUSSION TITLE
Collaboration with "shadcn_flutter" for one powerful shadcn

### DISCUSSION THREAD

**Discussion by davidjadczyk** [nank1ro/flutter-shadcn-ui#123] [General]
I just saw this fresh repo: https://github.com/sunarya-thito/shadcn_flutter 
It implemented also quite a few shadcn components. 
Maybe @nank1ro you could collaborate to coming to one and even more powerful shadcn package for us devs in the flutter community. 
I really appreciate this work, as it helps to quickly build really good looking applications!

**Comment by nank1ro** [comment]
My repo was created the 05/01/2024 while he started the 12/02/2024. He never contacted me to contribute.
It's an open source project, I'd love to have contributions.

Each widget I make takes some time because I try to solve problems in a simple way, making each widget extremely customizable.

Another library could probably come first with more widgets, but in the long run it's the quality the most important thing.
